packages/rx_bloc_cli/mason_templates/bricks/lib_auth_matrix/__brick__/lib/lib_auth_matrix/methods/pin_biometric/di/auth_matrix_pin_biometrics_page_with_dependencies.dart

```diff
@@ -0,0 +1,41 @@
+{{> licence.dart }}
+
+import 'package:flutter/cupertino.dart';
+
+import 'package:provider/provider.dart';
+import 'package:provider/single_child_widget.dart';
+
+import '../../../models/auth_matrix_response.dart';
+import '../services/auth_matrix_pincode_service.dart';
+import '../views/auth_matrix_pin_biometrics_page.dart';
+
+class AuthMatrixPinBiometricsPageWithDependencies extends StatelessWidget {
+  const AuthMatrixPinBiometricsPageWithDependencies({
+    required this.transactionId,
+    required this.authMatrixResponse,
+    super.key,
+  });
+
+  final AuthMatrixResponse authMatrixResponse;
+  final String transactionId;
+
+  @override
+  Widget build(BuildContext context) => MultiProvider(
+        providers: [
+          ..._services,
+        ],
+        child: AuthMatrixPinBiometricsPage(
+          transactionId: transactionId,
+        ),
+      );
+
+  // AuthMatrixPinCodeService
```

Comment:
This feels like a slightly redundant comment

Reply:
It has been removed